some infra tests need a connection to github to complete

```
    
>           raise ConnectionError(e, request=request)
E           requests.exceptions.ConnectionError: HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /compiler-explorer/compiler-explorer/main/etc/config/c%2B%2B.amazon.properties (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fa65e56d360>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))

.env/lib/python3.10/site-packages/requests/adapters.py:565: ConnectionError
=========================== short test summary info ============================
FAILED bin/test/amazon_properties_test.py::test_should_have_staticliblink - r...
FAILED bin/test/amazon_properties_test.py::test_googletest_should_have_versions
```

--- a/bin/test/amazon_properties_test.py
+++ b/bin/test/amazon_properties_test.py
@@ -1,5 +1,6 @@
 import logging
-
+import pytest
+import requests
 from lib.amazon_properties import get_properties_compilers_and_libraries, get_specific_library_version_details
 
 logger = logging.getLogger(__name__)
@@ -10,25 +11,33 @@ logger.level = 9
 #     assert len(_compilers) > 0
 #     assert len(_libraries) > 0
 
+
 def test_should_have_staticliblink():
-    [_compilers, _libraries] = get_properties_compilers_and_libraries('c++', logger)
-    assert 'googletest' in _libraries
-    assert len(_libraries['googletest']['versionprops']['trunk']['staticliblink']) > 0
-    assert _libraries['googletest']['versionprops']['trunk']['staticliblink'][0] == 'gtest'
-    assert _libraries['googletest']['versionprops']['trunk']['staticliblink'][1] == 'gmock'
+    try:
+        [_compilers, _libraries] = get_properties_compilers_and_libraries('c++', logger)
+        assert 'googletest' in _libraries
+        assert len(_libraries['googletest']['versionprops']['trunk']['staticliblink']) > 0
+        assert _libraries['googletest']['versionprops']['trunk']['staticliblink'][0] == 'gtest'
+        assert _libraries['googletest']['versionprops']['trunk']['staticliblink'][1] == 'gmock'
+    except requests.exceptions.ConnectionError:
+        pytest.skip("Connection error in test_should_have_staticliblink, which needs internet access")
+
 
 def test_googletest_should_have_versions():
-    [_compilers, _libraries] = get_properties_compilers_and_libraries('c++', logger)
-    assert 'googletest' in _libraries
-    assert len(_libraries['googletest']['versionprops']) > 0
-    assert _libraries['googletest']['versionprops']['110']['lookupversion'] == 'release-1.10.0'
-    assert _libraries['googletest']['versionprops']['110']['version'] == '1.10.0'
+    try:
+        [_compilers, _libraries] = get_properties_compilers_and_libraries('c++', logger)
+        assert 'googletest' in _libraries
+        assert len(_libraries['googletest']['versionprops']) > 0
+        assert _libraries['googletest']['versionprops']['110']['lookupversion'] == 'release-1.10.0'
+        assert _libraries['googletest']['versionprops']['110']['version'] == '1.10.0'
 
-    details = get_specific_library_version_details(_libraries, 'googletest', '1.10.0')
-    assert details != False
+        details = get_specific_library_version_details(_libraries, 'googletest', '1.10.0')
+        assert details != False
 
-    details = get_specific_library_version_details(_libraries, 'googletest', 'release-1.10.0')
-    assert details != False
+        details = get_specific_library_version_details(_libraries, 'googletest', 'release-1.10.0')
+        assert details != False
+    except requests.exceptions.ConnectionError:
+        pytest.skip("Connection error in test_googletest_should_have_versions, which needs internet access")
 
 # def test_should_not_contain_g412():
 #     [_compilers, _libraries] = get_properties_compilers_and_libraries('c++', logger)

